aten/src/ATen/function_wrapper.py

```diff
@@ -695,8 +695,14 @@ def to_return_type(arg, option):
     }
 
 
+def append_op_registration(op_registrations, option, registration):
+    # type: (List[Tuple[str, str]], FunctionOption, str) -> None
+    opname = option['schema_string'].split('.')[0].split('(')[0]
+    op_registrations.append((opname, registration))
```

Comment:
This looks a bit dodgy

Reply:
+1, getting an op name out of an option structure should be a canonical operation, and hopefully not one that involves slicing it out of the schema string. Is `option[api_name]` what's needed here? That's what I'd guess from scanning the code, but there might be some subtlety.

If we actually need new logic for getting this, ideally there would be a function `get_op_name(option)` that encapsulates the logic so it's obvious and easy to reuse.

Reply:
What I need is namespace + operator name, e.g.: "aten::conv2d". I did a quick search when I wrote this but couldn't find the namespace part in Declarations.yaml - turns out it's hard coded (always "aten::" for all registration generated by this codegen script): 
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native_parse.py#L393
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/function_wrapper.py#L165

I'll change it to "aten::${operator_name}" as other places in this script - could do some cleanup later to consolidate them all to a canonical function.

Reply:
> consolidate them all to a canonical function

Yeah, or even just a global constant `ATEN_NAMESPACE` or something. Thanks for running this down!